### PR TITLE
Map

### PR DESCRIPTION
**Pull request by truhin-andrei**

### CODE REVIEW

File: src/components/MapBlock/MapBlock.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-extra-parens */
+/* eslint-disable array-bracket-spacing */
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { Paper } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import { YMaps, Map, Placemark } from 'react-yandex-maps'
+import CircularProgress from '@material-ui/core/CircularProgress'
+
+import classNames from 'classnames'
```
Comment (rss-maksim) [inline src/components/MapBlock/MapBlock.jsx]: I'd recommend to reorder imports, for example classNames put up

File: src/components/MapBlock/MapBlock.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+}))
+const MapBlock = ({ coordinates, name, currentLang }) => {
+  const classes = useStyles()
+  const [loadingMap, setLoadingMap] = useState(true)
+  const loadClasses = classNames({
+    loadingMap,
+    noLoadingMap: !loadingMap
+  })
+  const localMap =
+    (currentLang === 'ru' && 'ru_RU') ||
```
Comment (rss-maksim) [inline src/components/MapBlock/MapBlock.jsx]: There is constants for language, you can use it

File: src/pages/director.jsx [hunk 3]
```diff
@@ -97,6 +99,11 @@ const DirectorPage = () => {
           className="director-info"
         >
           <TableOfWorks work={directorData.works} />
+          <MapBlock
+            coordinates={directorData.coordinates.split(',').map(item => +item)}
+            name={directorData.first_name + ' ' + directorData.last_name}
```
Comment (rss-maksim) [inline src/pages/director.jsx]: You can use template strings like this 
```
name={`${directorData.first_name} ${directorData.last_name}`}
```

**Review by rss-maksim — APPROVED** [review]

File: src/components/MapBlock/MapBlock.jsx (hunk 2 above)
Comment (truhin-andrei) [reply src/components/MapBlock/MapBlock.jsx]: done

File: src/pages/director.jsx (hunk 3 above)
Comment (truhin-andrei) [reply src/pages/director.jsx]: done

**Pull request merged**